test/Microsoft.AspNetCore.SignalR.Tests/HubEndpointTests.cs

```diff
@@ -540,7 +540,17 @@ public class HubEndpointTests
                 client.Dispose();
 
                 // Ensure the client channel is empty
-                Assert.Null(client.TryRead());
+                var message = client.TryRead();
+                switch (message)
+                {
+                    case CloseMessage close:
+                        break;
+                    case null:
```

Comment:
Can you just remove this case since it would be handled by the default case

Reply:
True. Will do.